scripts/parse_numeric_data.py

```diff
@@ -0,0 +1,151 @@
+import os
+import json
+from pprint import pprint
+import re
+
+SOURCE_PATH = "../raw_cldr_translation_data/"
+TARGET_PATH = "../number_parser/translation_data_merged"
+SUPPLEMENTARY_PATH = "../supplementary_translation_data/"
+
+REQUIRED_KEYS = ["spellout-cardinal", "spellout-numbering"]
+CAPTURE_BRACKET_CONTENT = r'\{(.*?)\}'
+
+LARGE_EXCEPTIONS = {}
+
+UNIT_NUMBERS = {}
+BASE_NUMBERS = {}
+MTENS = {}
+MHUNDREDS = {}
+MULTIPLIERS = {}
+VALID_TOKENS = {}
+
```

Comment:
there are a lot of things about this script that should be checked and fixed. I will wait until we clarify something.

Reply:
Sure, removed the script and merged data from this PR. (Now it only has the raw and supplementary json files). Will open new PRs for other changes.